### ISSUE TITLE
Research 'Support Vector Machines' (wiki)

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/machine-learning#1]
To gain a better understanding of [_Support Vector Machines_](http://en.wikipedia.org/wiki/Support_vector_machine), we will research the following:

**SVM Classification**
- General Definition
- Requirement
- Other Modeling Techniques (SVM in general)
- Limitations (curse of dimensionality, applies generally to multi-dimensional models)

**SVM Regression**
- Background: Regression Analysis
- SVM: Regression Analysis

**Loss Functions**
- Loss Functions: Quick Summary
- Loss Functions: SVM Regression
- Loss Functions: SVM Classification
- Types of Regression

**Comment by jeff1evesque** [comment]
https://bpaste.net/show/35fec1bfad0d

**Comment by jeff1evesque** [comment]
The following are examples of using `LabelEncoder` in sklearn:

```python
>>> le = preprocessing.LabelEncoder()
>>> le.fit(["paris", "paris", "tokyo", "amsterdam"])
LabelEncoder()
>>> list(le.classes_)
['amsterdam', 'paris', 'tokyo']
>>> le.transform(["tokyo", "tokyo", "paris"]) 
array([2, 2, 1]...)
>>> list(le.inverse_transform([2, 2, 1]))
['tokyo', 'tokyo', 'paris']
```
